BLSQ/iaso
# IA-3192 Update the payment lot XLS export

Update the payment lot XLS export.

This will allow to "make payments applicable to only certain change request types" (IA-3192).

Related JIRA tickets : [IA-3192](https://bluesquare.atlassian.net/browse/IA-3192)

## Changes

- new wording "Total Change Requests Count" instead of "Change Requests Count" for column `I`
- new column "Org Unit Creation Count"
- new column "Org Unit Change Count"
- new dynamic columns: payments are broken down by number of forms

A new `OrgUnitChangeRequest.field` is added to make it easier to know if a change request is related to an "Org Unit Creation" or an "Org Unit Change".

## How to test

1. ensure the `Payments` module is active
2. ensure your user has the `Payments page - Read & Write` permission
3. got to `/dashboard/payments/lots/`
4. click on one of the the rightmost arrow "Download payments"
5. check that the downloaded `.xls` has the number of changes of each form

![ui](https://github.com/user-attachments/assets/391b7ee6-d313-4ea1-8a41-6665ab844b0a)

## Print screen

![xls](https://github.com/user-attachments/assets/a77446d1-d144-435d-8cef-8f7b32a466b1)


[IA-3192]: https://bluesquare.atlassian.net/browse/IA-3192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
